src/SWP/Bundle/ContentBundle/Doctrine/ODM/PHPCR/Route.php

```diff
@@ -18,4 +18,45 @@
 
 class Route extends Page implements RouteInterface
 {
+    /**
+     * @var string
+     */
+    protected $templateName;
+
+    /**
+     * @var string
+     */
+    protected $type;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getTemplateName()
+    {
+        return $this->templateName;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function setTemplateName($templateName)
+    {
+        $this->templateName = $templateName;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getType()
```

Comment:
none of the interfaces is forcing to use that method, or am I missing something?